2-mandatory/1-recipes.js

```diff
@@ -22,4 +24,46 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code her
+const myRecipes = [
+  recipeOne = {
+    title: "avacado salad",
+    servings: 3,
+    ingredients: ["avacado", "cucumber", "mayonnesse", "boiled eggs"]
+  
+  },
+  recipeTwo = {
+    title: "chicken momo",
+    servings: 4,
+    ingredients: ["plain flour", "mince", "onion", "msg", "salt"]
+  
+  },
+  recipeThree = {
+    title: "fry rice",
+    servings: 1,
+    ingredients: ["rice", "vegetable oil", "green vegetables"]
+  
+  },
+  recipeFour = {
+    title: "Fish Curry",
+    servings: 3,
+    ingredients: ["fish", "tamaring", "mustard seed", "cumin", "corriander"]
+  
+  },
+  recipeFive = {
+    title: "biryani",
+    servings: 5,
+    ingredients: ["rice", "chicken", "cinnamon", "ginger-garlic"]
+  
+  }
+]
+for (let recipe of myRecipes) {
+  let title = recipe.title;
+  let serving = recipe.servings;
+  let ingredients = function() {for (let ingredient of recipe.ingredients) {console.log(ingredient)}};
+  console.log(title)
+  console.log("Serves:", serving)
+  console.log("Ingredients:", ingredients())
```

Comment:
When running this function, the output for each of the recipes looks like this 
ingredient,
ingredient,
ingredient,
Ingredients: undefined
Can you see why that might be? The ingredients function is working to return the list of the ingredients but something is going wrong, can you see what it might be?

Reply:
I assume I need to add 'return' in line 63 to return the function output ?